<ui version="4.0" >
 <class>dlgCheckConfig</class>
 <widget class="QDialog" name="dlgCheckConfig" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>152</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Bottle Checker</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>370</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;b>Windows Version:&lt;b></string>
   </property>
  </widget>
  <widget class="QLabel" name="label2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;b>Graphics Memory (MB):&lt;b></string>
   </property>
  </widget>
  <widget class="QLabel" name="label3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;b>Visual C++ 2005 run-time:&lt;b></string>
   </property>
  </widget>
  <widget class="QLabel" name="lblWinVersion" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>wine default</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMemory" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>40</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>auto detect</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVCRun" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>70</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>not installed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgCheckConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgCheckConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
